src/scss/variables.scss

```diff
@@ -246,8 +246,8 @@ $themes: (
         badgePrimaryBackground: $darkLinks,
         badgePrimaryBackgroundHover: $darkPrimary-alt,
         badgePrimaryText: $darkDarkBlue2,
-        badgeSecondaryBackground: $darkDarkBlue2,
-        badgeSecondaryText: $white,
+        badgeSecondaryBackground: $secondary,
+        badgeSecondaryText: #212529,
```

Comment:
These values for the badge background and text are different to what Danielle requested [here](https://app.asana.com/0/0/1201295377550891/1201307132191437/f), is that intentional?

Reply:
Ah I missed that update, I'll get it fixed!

Reply:
@eliykat Fixed!

![Screen Shot 2021-11-03 at 9 26 52 PM](https://user-images.githubusercontent.com/8926729/140242057-30f5bbfb-04f3-41cf-8a25-21f05ea71bb2.png)